src/theme/utils.ts

```diff
@@ -86,7 +86,28 @@ export enum FigmaTokenValueType {
   Pixels = 'pixels',
   String = 'string',
   Number = 'number',
+  BoxShadow = 'boxShadow',
 }
+
+/** The type of the object inside the theme/constants folder  */
+export type FigmaTokensObject<T extends string | symbol> = Record<
```

Comment:
i only have some concerns about this part here and if this is correct, I will have a look as well